### ISSUE TITLE
Question about the `pool` features

### ISSUE THREAD

**Issue by FunkyKoki** [vitoralbiero/img2pose#52]
Thanks again for your work.

As I see from the code, the backbone network in your original implementation outputs feature maps in _**5**_ scales, and they are stored by an OrderedDict with the `keys` = '0', '1', '2', '3', 'pool'. 
However, in the second phrase, the `MultiScaleRoIAlign` only uses `featmap_names=["0", "1", "2", "3"]`. 

So, why drop the `pool` features, since they are also supervised by the loss function?

**Comment by vitoralbiero** [comment]
You could keep the `pool` features, but since we are using the standard Faster R-CNN implementation, we just kept what was defined [there](https://github.com/pytorch/vision/blob/a2f65656b2b9dd7cc4363c8ad4afcbf89a25952b/torchvision/models/detection/faster_rcnn.py#L226).
I suspect the performance won't change much by adding it though.

Hope this helps.

**Comment by FunkyKoki** [comment]
Thank you for your answer.
